src/main/java/com/rk/rentacarapp/utils/JwtUtil.java

```diff
@@ -0,0 +1,144 @@
+package com.rk.rentacarapp.utils;
+
+import com.rk.rentacarapp.model.User;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+
+@Component
+public class JwtUtil {
+
+    private String APP_NAME = "Rent a car application";
+
+    public String SECRET = "ourLittleSecret, psst!";
+
+    private int EXPIRES_IN = 180; //Value represents minutes (so.. 3h)
```

Comment:
This should be set in milliseconds, the comment does mention that it's in minutes but what I'd ask myself is why? The main usage seems to be the one below where you're converting it to milliseconds, so why not just store it in milliseconds right away?

Reply:
The name could be TOKEN_EXPIRATION_TIME